Userland/Libraries/LibWeb/HTML/Window.cpp

```diff
@@ -297,6 +298,134 @@ static bool check_if_a_popup_window_is_requested(OrderedHashMap<String, String>
     return true;
 }
 
+// FIXME: This is based on the old 'browsing context' concept, which was replaced with 'navigable'
+// https://html.spec.whatwg.org/multipage/window-object.html#window-open-steps
+JS::ThrowCompletionOr<HTML::WindowProxy*> Window::open_impl(StringView url, StringView target, StringView features)
```

Comment:
Any reason why this can't be a `WebIDL::ExceptionOr`?

Reply:
(I'd prefer to convert to `ThrowCompletionOr` as late as possible, i.e. at the call site)

Reply:
The only other impl method that could throw exceptions was document_tree_child_browsing_context_count, and that one used ThrowCompletionOr, so I did as well. But you're right, it's nicer this way, fixed.